Deceive/Utils.cs

```diff
@@ -97,20 +70,14 @@ public static string GetLCUPath()
                 path = File.ReadAllText(CONFIG_PATH);
             else
             {
-                object registry = Registry.GetValue("HKEY_CURRENT_USER\\Software\\Riot Games\\RADS", "LocalRootFolder", "");
-                object registryNew = Registry.GetValue("HKEY_LOCAL_MACHINE\\SOFTWARE\\WOW6432Node\\Riot Games, Inc\\League of Legends", "Location", "");
-                if (registry == null && registryNew == null)
+                object registry = Registry.GetValue("HKEY_LOCAL_MACHINE\\SOFTWARE\\WOW6432Node\\Riot Games, Inc\\League of Legends", "Location", "");
```

Comment:
Do you know if older League installs also write this registry key? Otherwise it may be good to have the old value as a fallback.

Reply:
I think only the 'new' installer does. It's not that new though, it already existed while RADS was still in use and it only wrote to the new registry key. 
But older League installs might still have the old one... Then again I never had that one and if it exists then it should only in 32-bit systems, since on 64-bit Windows as 32-bit program it should be under 'WOW6432Node' I think.

Reply:
Ah it's fine. They can manually select their install location anyway.